### PR TITLE
improvement: S3C-4272 add optional duration seconds param

### PR DESCRIPTION
**Pull request by ilkescality**
Adds optional `durationSeconds` param to `generateAccountAccessKey` method.

### CODE REVIEW

**Review by saitharunthodupunuru — APPROVED** [review]
LGTM

File: bin/vaultclient [hunk 1]
```diff
@@ -128,10 +128,12 @@ program
     .option('--name <NAME>')
     .option('--accesskey <ExternalAccessKey>')
     .option('--secretkey <ExternalSecretKey>')
+    .option('--expirationTime <ExpirationTime>')
```
Comment (anurag4DSB) [inline bin/vaultclient]: Is `expirationTime` the target time when the keys are meant to expire or is it the duration in number of seconds?

**Review by anurag4DSB — CHANGES_REQUESTED** [review]
Needs a bit more clarity as to what is expected as input.
By the name, `expirationTime`, it's the time at which the keys will expire.

If that is correct, then is a user expected to input the time in epoch?

File: bin/vaultclient (hunk 1 above)
Comment (anurag4DSB) [reply bin/vaultclient]:
```suggestion
    .option('--expirationTime <ExpirationTime>', '<<some explanation of what type of input is expected by vaultclient>>', parseInt)
```

Comment (ilkescality) [reply bin/vaultclient]: It's the duration in number of seconds, just added the explanation.

**Review by anurag4DSB — APPROVED** [review]

**Pull request merged**